modules/serverless-customize-labels-annotations-routes.adoc

```diff
@@ -0,0 +1,53 @@
+[id="serverless-customize-labels-annotations-routes_{context}"]
+= Customizing labels and annotations for {product-title} routes
+
+{product-title} routes support the use of custom labels and annotations, which can be configured by modifying the `metadata` spec of a Knative service. Custom labels and annotations are propagated from the service to the Knative route, then to the Knative ingress, and finally to the {product-title} route.
+
+The following annotations are supported by Knative Serving:
+
+* Disabling routes (`DisableRouteAnnotation`), which is specified by the `serving.knative.openshift.io/disableRoute` annotation.
+* Enabling passthrough (`EnablePassthroughRouteAnnotation`), which is specified by the `serving.knative.openshift.io/enablePassthrough` annotation.
```

Comment:
Not sure why these have their camelcased constant definitions here. Users would never see/know/use them.

Reply:
Literally just because I goofed wrt. the example and thought I should include them, I'd much rather take them out, thanks!